### ISSUE TITLE
page 156 fix what?

### ISSUE THREAD

**Issue by Lairo** [head-first-csharp/fifth-edition#5]
The final question - "Can you sleuth out how to fix it?"

I do not understand what the question is asking, nothing is broken.

The question is just asking to think, perhaps a brain tag is more appropriate? (I mentally just decided to skip it and carry on, which I do not think the intention was.)

![image](https://github.com/user-attachments/assets/4427b941-c9f6-45d3-9f98-191f4df12476)

Btw do you mind if I ask such questions from a Readers point of view, as I continue throught out this book?

**Comment by andrewstellman** [comment]
Absolutely! We worked really hard to review everything in the book, but you've got a very good eye and are finding some of the last few rough edges. 

Could you do me a favor and start entering these issues as errata instead of raising GitHub issues—there's definitely no problem with GitHub issues 🙂 it's just a little easier for me to track on the errata page: https://app.oreilly.com/authors/portal.csp#manage-errata

**Comment by Lairo** [comment]
I think you have dropped me an author's link.
Try accessing this link with an incognito page on your browser.
You should see this too, I think.
Also I am not sure if I would be able to post there a lot as I only would have access via a Trial account for max 30 days, or perhaps I misunderstood and that part is free.

![image](https://github.com/user-attachments/assets/9307848e-a932-4cab-bbaf-74693adc68cc)

**Comment by andrewstellman** [comment]
Sorry about that. You can find the errata here: https://www.oreilly.com/catalog/errata.csp?isbn=0636920803027 – and there's a link at the top to add your own errata. I added errata for this issue already.
